com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/CreateAppStepBaseTest.java

```diff
@@ -0,0 +1,41 @@
+package com.sap.cloud.lm.sl.cf.process.steps;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.cloudfoundry.client.lib.domain.ServiceKey;
+
+import com.sap.cloud.lm.sl.cf.client.lib.domain.CloudApplicationExtended;
+import com.sap.cloud.lm.sl.cf.client.lib.domain.CloudServiceExtended;
+import com.sap.cloud.lm.sl.cf.core.cf.PlatformType;
+
+public abstract class CreateAppStepBaseTest extends SyncFlowableStepTest<CreateAppStep> {
+
+    protected StepInput stepInput;
+    protected CloudApplicationExtended application;
+    
+    protected abstract void validateClient();
```

Comment:
There's no reason to have a `protected abstract` method when it's not being used anywhere in the class. The child classes may not want to validate the interactions with the client. Let them do what they want. :)

Reply:
Okay. Done